Add community files & badges (#24)
talpalang/talpa: CODE_OF_CONDUCT.md

# Contributor Covenant Code of Conduct

## Our Pledge

> In the interest of fostering an open and welcoming environment, we as
> contributors and maintainers pledge to making participation in our project and
> our community a harassment-free experience for everyone, regardless of age, body
> size, disability, ethnicity, sex characteristics, gender identity and expression,
> level of experience, education, socio-economic status, nationality, personal
> appearance, race, religion, or sexual identity and orientation.

Reviewer: I don't think we have to fully write out what everyone means,  
If you need to write out what everyone means i think there has gone something wrong. **Everyone** means **Everyone** to me.  

```suggestion
In the interest of fostering an open and welcoming environment, we as
contributors and maintainers pledge to making participation in our project and
our community a harassment-free experience for everyone.
```

Author: This was taken from [here](https://www.contributor-covenant.org/) as it is an easy drop-in. I do think that your suggestion is definitely much more understandable. Especially for people new to open-source development.

## Our Standards

Examples of behavior that contributes to creating a positive environment
include:

* Using welcoming and inclusive language
* Being respectful of differing viewpoints and experiences
* Gracefully accepting constructive criticism

> * Focusing on what is best for the community

Reviewer: I think i care more about this project rather than the community :)
```suggestion
* Focusing on what is best for this project
```
We are not making a community we are making a project.  
Ya a community might be a result but the main focus will be this project.

threatening, offensive, or harmful.

## Scope

This Code of Conduct applies both within project spaces and in public spaces
when an individual is representing the project or its community. Examples of
representing a project or community include using an official project e-mail
address, posting via an official social media account, or acting as an appointed
representative at an online or offline event. Representation of a project may be
further defined and clarified by project maintainers.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported by contacting the project team at . All
complaints will be reviewed and investigated and will result in a response that
is deemed necessary and appropriate to the circumstances. The project team is
obligated to maintain confidentiality with regard to the reporter of an incident.
Further details of specific enforcement policies may be posted separately.

Project maintainers who do not follow or enforce the Code of Conduct in good
faith may face temporary or permanent repercussions as determined by other
members of the project's leadership.

> ## Attribution

Reviewer: If we apply the above we have to change the notations here
